api/src/pcapi/core/finance/models.py

```diff
@@ -172,7 +172,6 @@ class PricingLineCategory(enum.Enum):
     OFFERER_REVENUE = "offerer revenue"
     OFFERER_CONTRIBUTION = "offerer contribution"
     PASS_CULTURE_COMMISSION = "pass culture commission"
-    OFFERER_RETRIEVAL = "offerer retrieval"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏Comment ce sera qualifié alors ?

Reply:
J'y fais référence plus haut, l'intégration des incidents dans la facture est ni plus ni moins que l'inverse des montants déjà enregistrés de type `OFFERER_REVENUE` ou `OFFERER_CONTRIBUTION`. Ce nouveau type de ligne ajouté à priori n'a plus sa place ici